src/main/java/org/testeditor/web/xtext/index/FileBasedXtextIndexFiller.xtend

```diff
@@ -0,0 +1,24 @@
+package org.testeditor.web.xtext.index
+
+import java.io.File
+import org.eclipse.emf.common.util.URI
+
+class FileBasedXtextIndexFiller {
```

Comment:
Class without state, i.e. no fields & all methods operate only on their arguments.

One could argue that "filling" should be an operation the index itself should be responsible for.

Reply:
we'll have to look into this after the integration took place